jwst/extract_1d/extract_1d_step.py

```diff
@@ -186,6 +187,15 @@ def process(self, input):
         elif isinstance(input_model, ModelContainer):
             self.log.debug('Input is a ModelContainer')
         elif isinstance(input_model, datamodels.MultiSlitModel):
+            #  If input is a 3D rateints (which is unsupported) skip the step
+            try:
+                if len((input_model[0]).shape) == 3:
+                    self.log.warning('ext_1d_step step will be skipped as 3D input is unsupported')
```

Comment:
The log already tells you what step is being run, so this can just say "3D input is not supported; step will be skipped"